.github/workflows/fetch-and-ingest-genbank-master.yml

```diff
@@ -96,22 +100,10 @@ jobs:
           --memory 68GiB \
           --env AWS_ACCESS_KEY_ID \
           --env AWS_SECRET_ACCESS_KEY \
-          --env AWS_DEFAULT_REGION \
           --env GITHUB_RUN_ID \
           --env SLACK_TOKEN \
           --env SLACK_CHANNELS \
-          --env PAT_GITHUB_DISPATCH \
+          --env PAT_GITHUB_DISPATCH="$GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH" \
           . \
             --configfile config/genbank.yaml \
-            --config "${config[@]}"
-      env:
-        AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        PAT_GITHUB_DISPATCH: ${{ secrets.GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH }}
-        TRIAL_NAME: ${{ inputs.trial_name }}
-        FETCH_FROM_DATABASE: ${{ inputs.fetch_from_database != '' && inputs.fetch_from_database || true }}
-
-    - name: notify_pipeline_failed
-      if: ${{ failure() }}
-      run: ./vendored/notify-on-job-fail Ingest nextstrain/ncov-ingest
+            $CONFIG_OVERRIDES
```

Comment:
FWIW, this opens up a command-line injection via `inputs.trial_name` (though in practice it's probably fine given our usage).

Reply:
OH, I've repeated this pattern across other GH Action workflows 🙈 

What would be the safer thing to do here?

Reply:
The best thing is building up an array of strings, which can then be expanded with quoting to prevent subsequent word splitting. But with the config overrides being computed in a separate job, you can't pass an array, only a single long string. One way forward with that is to use shell-quoting semantics inside that single long string (i.e. nested quoting), but that gets tedious and is easy to mess up. Stepping back, why did the config override get moved into a separate job? Does it need to be?

Or, as an alternative to passing around a command-line snippet in a string/env var, we could produce another YAML file and pass that instead?

Reply:
> Stepping back, why did the config override get moved into a separate job? Does it need to be?

It had to get moved to a separate job because the config override is not currently part of the `pathogen-repo-build` workflow. Since configs are pretty different across pathogen workflows, I didn't want to shoehorn it into the reusable workflow. 

> Or, as an alternative to passing around a command-line snippet in a string/env var, we could produce another YAML file and pass that instead?

Ah, so build a custom config YAML that is uploaded/download as an artifact of the GH Action workflow. I think this is doable with updating `pathogen-repo-build` workflow to take a artifact name as an input and download the artifact as part of the reusable workflow. Yeah, this seems like the way to go if we are unable fully standardize configs across pathogens.

Reply:
> It had to get moved to a separate job because the config override is not currently part of the `pathogen-repo-build` workflow. Since configs are pretty different across pathogen workflows, I didn't want to shoehorn it into the reusable workflow.

Can't it go in the `run` input of the `pathogen-repo-build` workflow call?

Reply:
Hah, yes 🤦‍♀️